subsys/partition_manager/Kconfig

```diff
@@ -169,14 +169,6 @@ config PM_EXTERNAL_FLASH_BASE
 	hex "External flash base address"
 	default 0
 
-config PM_PARTITION_SIZE_MCUBOOT_SECONDARY
```

Comment:
Is it necessary to have a "migration period" where we keep this value, but deprecate it, stating that it is not used, and will be removed in the next release? IMO this is just pushing the work of un-setting the value forwards, and we might as well expose the users for this immediately.